### ISSUE TITLE
issue in Ubuntu 18.04.5 LTS 

### ISSUE THREAD

**Issue by RichardPinka** [anujdatar/onenote-desktop#13]
Hi, i installed your app in Ubuntu 18.04.5 LTS bionic beaver
and it is falling down.. i send yoy error message[](url)
![onenote eerror](https://user-images.githubusercontent.com/8267708/93085135-65ff1f80-f695-11ea-9090-26dd83de0fef.png)

**Comment by anujdatar** [comment]
I apologize for the massive delay in replying. I had fixed the error on my computer and moved on to other programs/projects since I initially published this package. I think i forgot to push my changes, before I changed my computer and lost some code.

And, I am not sure why I was unsubscribed from this repo's updates. I did not know so many people had the same errors, or that other people were even using this. An oversight on my part.

I have just published an updated version of the app, with Snap, Deb and AppImage files. I should have the snap package published in a bit.

While I labelled the release as alpha pre-release, I have tested it on Ubuntu/Kubuntu 20.04 and 20.10. Everything should be running fine. I will bump it to a stable release soon, once my laptop is fixed and have spent some time on further testing.

Thanks for the patience.
